src/views/Rewards/Rewards.js

```diff
@@ -58,14 +57,14 @@ export default function Rewards(props) {
 
   // If the full data has not been loaded, use current week data
   const weeksRewardsData = useMemo(() => {
-    if (allweeksRewardsData && selectedWeek) {
```

Comment:
Thought in camelcase that all and weeks were two separate words, but just nitpicking. I know you picked that up in the prev PR but all good if you think weeks should be lowercase

Reply:
hmmm, you mean they arent or are two seperate words. If so then camelCase would be allWeeks right? if its considered one word then it will be allweeks?

Reply:
that's what I mean - I take it as two words and would do it as allWeeks. but I'm fine either way